app/assets/javascripts/utilities/showUserAlertModal.js.erb

```diff
@@ -76,10 +75,23 @@ const getModalHtml = (
             </svg>
           </button>
       </div>
-      <div class="crayons-modal__box__body">
-        <p>${text}</p>
-        </br>
-        <button class="crayons-btn crayons-btn--icon" type="button" onClick="toggleUserAlertModal();">${confirm_text}</button>
+      <div class="crayons-modal__box__body pt-0 flex gap-2">
+        <div class="w-75">
+          <h2>
+            ${title}
+          </h2>
+          <p class="color-base-70">
+            ${text}
+          </p>
+          <button class="crayons-btn mt-4" type="button" onClick="toggleUserAlertModal();">
+            ${confirm_text}
+          </button>
+        </div>
+        <% if SiteConfig.mascot_image_url.present? %>
+          <figure class="w-25">
+            <img src="<%= SiteConfig.mascot_image_url %>" class="w-100" />
+          </figure>
+        <% end %>
```

Comment:
Talking with @maestromac, he figured out that the call to `SiteConfig` is breaking the CI, since `SiteConfig` is not available when the assets are precompiled. We might need a different approach here unfortunately :(

I went down the rabbit hole a bit here and it looks like this modal only triggers for rate limits. I guess we could add the `SiteConfig.mascot_image_url` to the comment page and then do:
```
<figure class="w-25">
  <img src="${document.getElementById('mascot-image').data.url}" class="w-100" />
</figure>
```

or something like that. We would probably need to extract the conditional into a different JS function too.